CMakeLists.txt

```diff
@@ -28,10 +28,12 @@ add_subdirectory(${CMAKE_CURRENT_SOURCE_DIR}/thirdparty)
 
 # Shady *Core*
 add_subdirectory(${CMAKE_CURRENT_SOURCE_DIR}/src/Core)
-# Shady *Cli*
-add_subdirectory(${CMAKE_CURRENT_SOURCE_DIR}/src/Cli)
-# Shady *Dnd*
-add_subdirectory(${CMAKE_CURRENT_SOURCE_DIR}/src/Dnd)
+if(SHADY_ENABLE_EXES)
```

Comment:
I will change to SHADY_ENABLE_EXECUTABLE and set the default to true on my side, after the merge